### PR TITLE
Adding super() examples and other minor fixes

### PR DESCRIPTION
**Pull request by migueleliasweb**
https://www.youtube.com/watch?v=EiOglTERPEo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ ADD Pipfile Pipfile.lock ./
 
 RUN pipenv install --deploy --system
 
-ENTRYPOINT ["ipython"]
+CMD ["ipython"]
```
Comment (lmagalhaes) [inline Dockerfile]: What is the difference?

File: README.md [hunk 2]
```diff
@@ -9,49 +9,53 @@
 
 docker-compose run --rm pycourse
 
-##Main topics:
-
 ### What is python
+* Brief python backstory
```
Comment (lmagalhaes) [inline README.md:13]: maybe it is a good idea to tell that you can see the Zen of Python by typing ```import this``` on python interactive shell :)
```python3
$ ipython
Python 3.6.5 (default, Mar 30 2018, 06:41:53)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
... 
```

**Review by lmagalhaes — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (migueleliasweb) [reply README.md:13]: I have completely forgotten about this! I think we must mention this but keeping it on the list makes it more visible ;)

File: Dockerfile (hunk 1 above)
Comment (migueleliasweb) [reply Dockerfile]: Just to make it easier to override when needed. If we use `ENTRYPOINT` the user will have to override with the `--entrypoint` flag but if we use the CMD the user will be able just to pass the executable path to the run command.

**Pull request merged**